<ui version="4.0" >
 <class>STSourceEditor</class>
 <widget class="QWidget" name="STSourceEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ST Source Editor</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="header_tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>418</width>
        <height>343</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Header</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="DefaultEditorWindow" native="1" name="header" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="source_tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>418</width>
        <height>343</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Source</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="margin" >
        <number>0</number>
       </property>
       <item>
        <widget class="DefaultEditorWindow" native="1" name="source" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="class_box" />
      </item>
      <item>
       <widget class="QToolButton" name="save_changes_button" >
        <property name="font" >
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text" >
         <string>...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="reload_button" >
        <property name="text" >
         <string>Reload from disk</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DefaultEditorWindow</class>
   <extends>QWidget</extends>
   <header>defaulteditorwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
